chamber/models/patch.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import unicode_literals
+
+from django.db.models.fields import Field
+
+
+def field_init(self, *args, **kwargs):
+    """
+    Patches a Django Field's `__init__` method for easier usage of optional `kwargs`. It defines a `humanized` attribute
+    on a field for better display of its value.
+    """
+    humanize_func = kwargs.pop('humanized', None)
+    if humanize_func:
+        def humanize(val, inst, *args, **kwargs):
+            return humanize_func(val, inst, field=self, *args, **kwargs)
+        self.humanized = humanize
+    else:
+        self.humanized = self.default_humanized
+    getattr(self, '_init_chamber_patch_')(*args, **kwargs)
+
+
+Field.default_humanized = None
+setattr(Field, '_init_chamber_patch_', Field.__init__)
```

Comment:
osobně bych raději dal vypnutí pep pro tento řádek. Dva důvody
1. označením řádku aby se neoznačil jako nevalidní říkáš, že víš co děláš
2. setattr mi přijde jako mnohem větší zásah než Field._init_chamber_patch_

stejně tak bych to aplikoval s _meta. Přeci nebudeme stále používat getattr. Nehledě na tom že getattr dělá o několik operací více než použití tečky v případě meta už by to mohlo být znát např. když budeme generovat nějaký velký export a pro každý řádek se něco z meta atributů vytáhne. Hledal jsem proč je vlastně meta protected. Zdá se že je to z historických důvodů kdy bylo api uzavřené a nezdokumentované.